opencog/atoms/pattern/PatternLink.cc

```diff
@@ -867,43 +862,18 @@ void PatternLink::make_term_tree_recursive(const Handle& root,
                                            Handle h,
                                            PatternTermPtr& parent)
 {
-	Type t = h->getType();
-
-	// If the pattern link is a quote, then we compare the quoted
-	// contents. This is done recursively, of course.  The QuoteLink
-	// must have only one child; anything else beyond that is ignored
-	// (as its not clear what else could possibly be done).
-	//
-	// Ignore quoting and unquoting nodes in the PatternTerm
-	if ((not parent->isQuoted() and (QUOTE_LINK == t or LOCAL_QUOTE_LINK == t))
-	    or (parent->getQuotationLevel() == 1
-	        and not parent->isLocalQuoted() and UNQUOTE_LINK == t)) {
-		if (1 != h->getArity())
-			throw InvalidParamException(TRACE_INFO,
-			                            "QuoteLink/UnquoteLink has "
-			                            "unexpected arity!");
-		h = h->getOutgoingAtom(0);
```

Comment:
Never mind. I spoke too soon. I see that it was moved to the PTM. That is.. unexpected, but I guess its OK.